WrightTools/kit/_array.py

```diff
@@ -433,3 +434,21 @@ def enforce_mask_shape(mask, shape):
     """
     red = tuple([i for i in range(len(shape)) if shape[i] == 1])
     return mask.max(axis=red, keepdims=True)
+
+
+def signed(d0):
+    """Tells bluesky whether to sign data. We have a 7.5 % tolerance right now.%"""
```

Comment:
The real comparison we want to make for this decision is to compare min and max to noise levels.  It would be nice to extract an estimate of noise for this evaluation.  But this is probably good enough for now.

Reply:
Just added some parameters, hopefully they look good

As to the second point: agreed! I think when we have more data it will be easier to obtain a better bound